docs/.custom_wordlist.txt

```diff
@@ -45,3 +45,25 @@ VMs
 VPN
 websockets
 yaml
+STSStack
+sshuttle
+vpn
+behavior
```

Comment:
Please remove this; it's mispelled for british english and not required here anyway.

```suggestion
```